data/sql/updates/pending_db_world/rev_1631940505188780442.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1631940505188780442');
+
+-- Deletes Black Lotus from all NPC loot tables
+DELETE FROM `creature_loot_template` WHERE `item` = 13468 AND `comment` LIKE '%Black Lotus%';
```

Comment:
what about adding and entry != 11447? 
https://tbc.wowhead.com/npc=11447/mushgog

Reply:
That's because Mushgog doesn't drop Black Lotus on AC right now, so there's no need to exclude him.

Also, I have some doubts about the Wowhead page that says he does, as the arena he spawns in (in Dire Maul) also has a Black Lotus spawn point that he may be linked to. So it seems likely that the Wowhead client just got confused between someone picking the herb and looting his corpse and erroneously linked the two. But that's supposition on my part.